### PR TITLE
Source oploverz

### PR DESCRIPTION
**Pull request by MoonLGH**
Added New Source, Oploverz - ID

also fixed some of comment typo on otakudesu comment (i dont think it need a version update, its only comment fixes(commit 2) )

i dont know why is there 2 commit here, idk how to remove the first commit, but the first commit its same as yesterday commit

### CODE REVIEW

File: src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                else -> "Unknown Resolution"
+            }
+        }
+        return Video(url, quality, url, null)
+    }
+
+    override fun videoListSelector(): String = "div.mctnx > div > div > a:nth-child(3)"
+
+    override fun videoUrlParse(document: Document) = throw Exception("not used")
+
```
Comment (jmir1) [inline src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt:149]: you need to override the function `List<Video>.sort()` in order for your quality preference to work.
example: https://github.com/jmir1/aniyomi-extensions/blob/9f791a0684940c40d8c8a6ddaaec5db29a0aac1e/src/en/gogoanime/src/eu/kanade/tachiyomi/animeextension/en/gogoanime/GogoAnime.kt#L175-L191

**Review by jmir1 — CHANGES_REQUESTED** [review]

File: src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt (hunk 1 above)
Comment (MoonLGH) [reply src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt:149]: Uh thats why my resolution didnt sorted correctly

File: src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt [hunk 2]
```diff
@@ -147,6 +149,39 @@ class Oploverz : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
     override fun videoUrlParse(document: Document) = throw Exception("not used")
 
+    private fun Elements.ordered(): Elements {
```
Comment (jmir1) [inline src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt:152]: isnt this completely useless?
you are adding at the end of the list so this doesnt change the list at all
just remove it, no?

**Review by jmir1 — CHANGES_REQUESTED** [review]

File: src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt (hunk 2 above)
Comment (MoonLGH) [reply src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt:152]: Uhhh suree

Comment (MoonLGH) [reply src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt:152]: Im coping the otakudesu list sorter though

Comment (MoonLGH) [reply src/id/oploverz/src/eu/kanade/tachiyomi/animeextension/id/oploverz/Oploverz.kt]: But sure ig

**Pull request merged**